src/DefaultBuilder/src/WebApplication.cs

```diff
@@ -27,7 +27,7 @@ public sealed class WebApplication : IHost, IApplicationBuilder, IEndpointRouteB
     internal WebApplication(IHost host)
     {
         _host = host;
-        ApplicationBuilder = new ApplicationBuilder(host.Services);
+        ApplicationBuilder = new ApplicationBuilder(host.Services, this.ServerFeatures);
```

Comment:
Style: Avoid `this.` unless it's required. You don't need it here do you?
```suggestion
        ApplicationBuilder = new ApplicationBuilder(host.Services, ServerFeatures);
```